doc/source/data/overview.rst

```diff
@@ -107,6 +107,12 @@ Preprocessing and ingest for ML training
 ----------------------------------------
 
 Use Ray Data to load and preprocess data for distributed :ref:`ML training pipelines <train-docs>` in a streaming fashion.
+Key supported features for distributed training include:
+
+- Support for fast out-of-memory recovery
+- Support for heterogeneous clusters
+- No dropped rows in distributed training (with :meth:`Dataset.streaming_split <ray.data.Dataset.streaming_split>`)
```

Comment:
We don't need disclose the detail of `(with :meth:Dataset.streaming_split <ray.data.Dataset.streaming_split>)` on front page I think.